src/pe_reports/data/db_query.py

```diff
@@ -98,6 +96,47 @@ def query_creds_view(org_uid, start_date, end_date):
             close(conn)
 
 
+def query_credsbyday_view(org_uid, start_date, end_date):
+    """Query credentials by date view ."""
```

Comment:
Remove space after view. Also this Docstring is the same as the function below, probably needs more description.

Reply:
https://github.com/cisagov/pe-reports/pull/191/commits/5ac357d060d21a31735c8f70c68bc4041e2bfe7c